### PR TITLE
initial draft

### PR DESCRIPTION
**Pull request by jpknwls**
## What does this PR do?
- opens Weekly Release draft

### Merge requirements satisfied

> For more information, visit the [Contributing Guide for Documentation](https://wiki.dendron.so/notes/b58801fc-43a9-4d42-a58b-eabc3e8538cb/)

- [ ] Check rendered output to ensure formatting is correct for renderings of wikilinks, note refs, tables, and images. `Dendron: Show Preview` can be used in your workspace to confirm the page renders as expected. Sometimes, `Dendron: Reload Index` needs to be ran if certain wikilinks aren't working as expected. (ignore this if contribution is made via the `Edit this page on GitHub` button from the published site)
- [ ] If this PR includes any refactoring (renaming notes, renaming headers, etc.), make sure that these changes were done via [Dendron Refactoring Commands](https://wiki.dendron.so/notes/srajljj10V2dl19nCSFiC/). This will ensure that any other impacted areas of the documentation, that link to or embed modified notes (via note references), are automatically updated.

Verify GitHub Actions tests are passing, which can be seen in the `Checks` tab of the PR:

- [ ] `URL validator` GitHub Action passes
- [ ] `Dendron site build` GitHub Action passes

> **NOTE:** If you are a new contributor, a maintainer will need to provide approval for GitHub Actions to run on your PRs.

### CODE REVIEW

File: vault/changelog.md [hunk 1]
```diff
@@ -7,6 +7,23 @@ created: 1601508213606
...
+- enhance(workspace): initialize workspace can create self contained vaults ([[docs|dendron://dendron.dendron-site/dendron.topic.vaults.self-contained##configuration]]) (#2569) @kaan
+- enhance(common): don't reload workspace when running doctor airtable command (#2620) @kevin
+
+### Fix
+- fix(workspace): fix dropped keystrokes issue in lookup (#2626) @jonathan
+- fix(internal): Engine updateNote not properly firing update events (#2622) @tuling
+- fix(markdown): support parenthesis in the image URL (#2634) @kaan
+- fix(views): Backlinks will no longer disappear in preview upon editing (#2608) @tuling
+- fix(lookup): autocomplete causes notes to be created in wrong vault  (#2623) @kevin
+- fix(views): Pass in a port-forwarded URL to preview for remote workspaces (#2624) @kaan
```
Comment (SeriousBug) [inline vault/changelog.md:25]: Could we reword this as "Preview now works in remote workspaces" or something similar, since that's the end user result?

Comment (jpknwls) [reply vault/changelog.md:25]: Yep, fixing now

File: vault/changelog.release.2022-03-29.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+<!-- [[Release Highlights|dendron://dendron.handbook/area.product.sop.release-highlights]] -->
+
+**Self Contained Vaults:** Description here.
+
+- More information: ([[docs|dendron://dendron.dendron-site/dendron.topic.vaults.self-contained##configuration]]) 
+
+**Deprecation Notice:**
+- Tree view V2 
+    - We will be focusing on improving Tree view V1 moving forward and will be removing V2. We will give a week's notice prior to removal.
```
Comment (kevinslin) [inline vault/changelog.release.2022-03-29.md:19]: remove the week notice remark here as well

File: vault/changelog.release.2022-03-29.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+<!-- TODO: Delete this section if not appliacble -->
+
+### Starboard and TIL Highlights
+<!-- TODO: update links. Delete section is no new items-->
+> These are highlights from the [[Dendron Discord|dendron://dendron.dendron-site/community.discord.channels]] `#starboard` and `#today-i-learned` channels.
+
+- ⭐ `aleksey#5276` shared a link to [a tool for finding out the size of Github repos](https://citizendot.github.io/gh-info/): _"A handy app to assess the size of a repo before cloning"_
+- 💡 `aleksey#5276` also shared [a way to write Latex mathematical equations in all Markdown editors](latex.codecogs.com): _"TIL you can write equations in Markdown even if your md editor doesn't support Latex:"_
+
+- 💡 `icanteven#0264` shared a link to [the Github changelog](): _"TIL I should be paying more attention to the GitHub Changelog. You can now follow organizations, and a new dashboard feed tab is in public beta and viewable now. I like it better than the current feed."_
```
Comment (ScriptAutomate) [inline vault/changelog.release.2022-03-29.md:47]:
```suggestion
- 💡 `icanteven#0264` shared a link to [the Github changelog](): _"TIL I should be paying more attention to the GitHub Changelog. You can now follow organizations, and a new dashboard feed tab is in public beta and viewable now. I like it better than the current feed."_

```

File: vault/changelog.release.2022-03-29.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+- fix(internal): Engine `updateNote` not properly firing update events
+- fix(markdown): support parenthesis in the image URL
+- fix(views): Backlinks will no longer disappear in preview upon editing
+- fix(lookup): autocomplete causes notes to be created in wrong vault
+- fix(views): Preview now works in remote workspaces
+
+## Community
+
+### General Updates
+<!-- TODO: Delete this section if not appliacble -->
```
[the remark this answers is not in the document]
Comment (jpknwls) [reply vault/changelog.release.2022-03-29.md:38]: where can they leave "feedback"?

[the remark this answers is not in the document]
Comment (kevinslin) [reply vault/changelog.release.2022-03-29.md:40]: in the #feedback channel in discord

**Review by kevinslin — APPROVED** [review]
